CONTRIBUTING.rst

```diff
@@ -64,32 +64,38 @@ Ready to contribute? Here's how to set up `fury` for local development.
 3. Add a tracking branch which can always have the last version of `fury`::
```

Comment:
`fury` -> FURY (?)

In other places, the term FURY has been used, or just fury, or Fury (without any markup). IMHO, it would be good to stick to the same naming.